### DISCUSSION TITLE
2022-02-15 Weekly Meeting 43 Agenda

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#181] [Weekly Meetings]
Weekly Meeting 43 is happening on Wednesday Feb 15th, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Announcements and Updates
- The [website](https://omigroup.org) is live!
- https://github.com/omigroup/omigroup/pull/180

Working Group Updates:
-  OMI Website update
   - Interoperator group blocking "friends of OMI section"
- OMI-Chan group
- WASM group
  - Experiment project milestone 1 pushed back https://github.com/omigroup/omi-wasm-group/discussions/19
  - Progress being made on Three.js, Unity, Godot, and Rust!
- gLTF group
  - https://github.com/omigroup/gltf-extensions/discussions/60
  - https://github.com/omigroup/gltf-extensions/pull/61
- any others?

Housekeeping:
- https://github.com/omigroup/omigroup/pull/182

Open Floor!
 
Please leave any additional agenda items below in the comments. Thanks!

**Comment by mrmetaverse** [comment]
New website is live! check it out at https://omigroup.org/ - great work from the OMI Website working group. This group meets every other Tuesday at 22:30pm UTC

**Comment by mrmetaverse** [comment]
RE: [Proposal to revise specifications section of our charter. 
](https://github.com/omigroup/omigroup/pull/180)
This is a broader change. There are some groups that are in fact creating standards, working in such a way that W3C requires signing the contributor license agreement through W3C. This signs over the patents or IP to W3C as an organization. This helps ensure that a contributor can't go back on their contributions later, ripping out the spec from underneath you after you have built on top of it. This is very important for the specification groups. In short, this does require that an individual use their "real name." This can pose as a challenge to many contributors to OMI overall. So if your group is NOT producing a specification or standard, it may not require that you sign the contributor license agreement. At the working group level, based on which type of work they are doing, it is encouraged that you have this discussion up-front and publish in your readme / working agreement if your group intends to recommend specifications which would require the signing of this agreement.

**Reply by mrmetaverse** [reply]
We must acknowledge that this is backfilling a policy. We have some who may have been participating in working groups who may not want to disclose their "real name," and we wish we had this discussion up-front explicitly for all our groups. Groups like WASM and gLTF working groups are working on specs/standards so we would ask that everyone contributing in these groups, and groups like these, sign the CLA. Apologies to anyone who may be negatively impacted by this policy change. 

We will take a week to discuss and vote next week. :calendar:

**Comment by mrmetaverse** [comment]
Reminder, we created the OMI-Interoperator group (or at least voted to ratfiy the group) to conduct discovery and due diligence around what a partnership or affiliate program could look like with OMI. We want to ensure that we give this group time to form, and kick off, and be mindful not to step on their toes.

**Reply by mrmetaverse** [reply]
group could meet as early as tomorrow if people have time.

**Comment by mrmetaverse** [comment]
OMI-Chan group: 
- need to have a budget discussion 
- our budget is steadily growing and we feel this group could be a great place to allocate some budget towards things like art, avatars, etc. 
- What is expected from this group? Then we can meet to discuss allocation of some of our budget.

**Comment by mrmetaverse** [comment]
gLTF group: 
- omi audio emitter moves to phase 3, we have enough implementers now 
- antpb created a 3ov (three object viewer) component for Wordpress! It implements the omi-audio emitter as well!

**Reply by mrmetaverse** [reply]
something else is moving into phase 2*

**Comment by mrmetaverse** [comment]
WASM group: 

Experiment project milestone 1 pushed back https://github.com/omigroup/omi-wasm-group/discussions/19

Progress being made on Three.js, Unity, Godot, and Rust!

**Comment by mrmetaverse** [comment]
Housekeeping: Merging the changes to readme. Removing "next steps" section from our initial kickoff.

**Comment by mrmetaverse** [comment]
We need to continue our discussion about the future structure of OMI leadership and governance. 

We are tracking this effort with this milestone: https://github.com/omigroup/omigroup/milestone/2

**Comment by mrmetaverse** [comment]
Metatraversal could possibly use some guidance

**Comment by mrmetaverse** [comment]
What about another demo night? And have some time for a meta-traversal bit to raise more awareness around the convo. We also have multiple people in our community who have been demoing awesome open metaverse work lately. Antpb said we can count them in to speak.

**Comment by mrmetaverse** [comment]
What about another game-night field-trip day?

**Reply by mrmetaverse** [reply]
suggestion: maybe use something that doesn't require a headset, but makes headset optional. i.e. explore some browser-based worlds like https://mementos.city/

**Comment by mrmetaverse** [comment]
Suggestion: What about making a OMI place to meet work and hack? This was previously on our backlog, and antpb offered to resume the work. 

OMI Space to meet, work, and play #44

**Reply by mrmetaverse** [reply]
Created this Milestone: https://github.com/omigroup/website/milestones?with_issues=no
Revived #44 and moved it to the website backlog: https://github.com/omigroup/website/issues/51

**Reply by mrmetaverse** [reply]
assigned to @antpb
